### PR TITLE
Feature/create configuration

### PR DESCRIPTION
**Pull request by artjomka**

### CODE REVIEW

File: src/commands/veloce/createconfig.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        {id: 'VELOCPQ__Value__c', title: 'VELOCPQ__Value__c'}],
+      path: outputfile
+    });
+    console.log('Using csv template', (csvTemplate));
+    const result = [];
+    fs.readdirSync(inputdir).forEach(file => {
+      console.log('processing configuration file:', file);
+      const output = {...csvTemplate[0]};
+      const {value} = JSON.parse(fs.readFileSync(inputdir + file, 'UTF-8').toString());
+      output.VELOCPQ__Value__c = value;
```
[the remark this answers is not in the document]
Comment (veloceapps) [reply src/commands/veloce/createconfig.ts:61]: how salesforce IDs gonna be created?
this wont load into sf

File: src/commands/veloce/createconfig.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  protected static requiresUsername = false;
+
+  // Comment this out if your command does not support a hub org username
+  protected static supportsDevhubUsername = false;
+
+  // Set this to true if your command requires a project workspace; 'requiresProject' is false by default
+  protected static requiresProject = false;
+
+  public async run(): Promise<AnyJson> {
+    const templateFile = this.flags.template || 'ConfigurationSetting_template.csv';
```
[the remark this answers is not in the document]
Comment (veloceapps) [reply src/commands/veloce/createconfig.ts:44]: arg is required anyway

File: src/commands/veloce/createconfig.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    const inputdir = this.flags.inputdir || '.';
+    const outputfile = this.flags.outputfile || 'VELOCPQ__ConfigurationSetting__c.csv';
```
[the remark this answers is not in the document]
Comment (veloceapps) [reply src/commands/veloce/createconfig.ts:46]: arg is required anyway

File: src/commands/veloce/createconfig.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (veloceapps) [reply src/commands/veloce/createconfig.ts:61]: also external ID should be present for each csv record: VELOCPQ__RereferenceId__c, etc

File: src/commands/veloce/createconfig.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (artjomka) [reply src/commands/veloce/createconfig.ts]: Magical defaults removed

File: src/commands/veloce/createconfig.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (artjomka) [reply src/commands/veloce/createconfig.ts]: Magical defaults removed

File: src/commands/veloce/createconfig.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    '$ sfdx veloce:createconfig -i config . -n BOARDING -t ConfigurationSettings_template.csv -o VELOCPQ__ConfigurationSetting__c.csv'
+  ];
+
+  public static args = [{name: 'file'}];
+
+  protected static flagsConfig = {
+    modelname: flags.string({char: 'n', description: messages.getMessage('modelnameFlagDescription'), required: true}),
+    inputdir: flags.string({char: 'i', description: messages.getMessage('inputdirFlagDescription'), required: true}),
+    outputfile: flags.string({char: 'o', description: messages.getMessage('outputfileFlagDescription'), required: true}),
+    template: flags.string({char: 't', description: messages.getMessage('templatefileFlagDescription'), required: true})
```
Comment (veloceapps) [inline src/commands/veloce/createconfig.ts:31]: no need to have template, since action is rigit anyway

File: README.md [hunk 5]
```diff
@@ -107,7 +108,43 @@ EXAMPLE
...
+
+USAGE
+  $ sfdx veloce:createconfig -n <string> -i <string> -o <string> -t <string> [--json] [--loglevel 
+  trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
+
+OPTIONS
+  -i, --inputdir=inputdir                                                           (required) Input directory to read
+                                                                                    property files
+
+  -n, --modelname=modelname                                                         (required) Name of model (directory)
```
Comment (veloceapps) [inline README.md:128]: seems like copy/paste issue

**Pull request merged**